CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 ## [Unreleased]
 ### Added
 - metadata parameter to IssuingPurchase resource
+### Changed
+- update core version
```

Comment:
Changed update core version fica estranho

A ideia é levar o Changed em consideração que nem em cima:
Added metadata parameter to IssuingPurchase resource